example.py

```diff
@@ -1110,6 +1165,8 @@ def core(self, slothy):
 
         slothy.config = conf.copy()
 
+        if self.timeout is not None:
```

Comment:
@dop-amin Is this duplication deliberate?

Reply:
No, that was copy-pasta.